apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -109,9 +111,23 @@ export function ModuleFunction({
                     >
                         Execute
                     </Button>
-                    <div className={clsx('temp-ui-override', { connected })}>
-                        <WalletWrapper />
-                    </div>
+                    <ConnectButton
+                        connectText={
+                            <>
+                                Connect Wallet
+                                <ArrowRight
+                                    fill="currentColor"
+                                    className="-rotate-45"
+                                />
+                            </>
+                        }
+                        size="md"
+                        className={clsx(
+                            connected ? st.connected : st.connect,
```

Comment:
Can we use inline tailwind?

Reply:
I've used inline before but was't working properly when building in prod mode. Then moved it to css modules to apply `!important`. I'll see if I can do something better

Reply:
Done (inline and important)